### PR TITLE
Check it out

### PR DESCRIPTION
**Pull request by dennisbailey**
Password validation is still bonkers, but I did update the README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <title>Responsive Login Page</title>
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  
+  <body>
+    
+    <form id="login-form">
+      <h1>Login</h1>
+      Email<br>
+      <input type="email" name="email" placeholder="Enter your email"><br>
```
Comment (bwreid) [inline index.html:14]: I'd add `<label>` elements above these inputs with `for` attributes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      Password<br>
+      <input type="password" name="password" pattern=".{6,}" required placeholder="6 characters or more please"><br>
+      <input type="submit" id= "submit" value="Login">
+    </form>
+    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+    <script src="http://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js"></script>
```
Comment (bwreid) [inline index.html:21]: I think some of the weirdness is coming from this library. You shouldn't need this.

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  $(document).on('click', '#submit', function(){
+      // Prevent event submission
+      event.preventDefault();
+      
+      console.log($('#login-form').validate());
+      
+      if ($('form').valid()) {
+        $('form').hide();
+        $('body').append('<h3 class ="center">There are many responsive login pages out there, but this one\'s mine.</h3>');
```
Comment (bwreid) [inline js/main.js:12]: This exercise asks you to create an element in memory to add as a success message. A more explicit way of doing this might be:

``` javascript
$message = $("<h3 class='center'></h3>");
$message.text("There are many responsive login pages out there, but this one's mine.");
$('body').append($message);
```

File: readme.md [hunk 4]
```diff
@@ -19,6 +19,13 @@ You will be building a responsive login page based on the mockup found in the [m
 
 __Discuss the relationship between HTML, JavaScript, the DOM API, and jQuery:__
 > Write your answer here...
+HTML provides the structure of a web page. Think of HTML as the skeleton that everything else attaches to
+
+CSS takes the bones of the HTML and adds the look and feel. CSS can be as simple as colors and shapes or as complex as animation. CSS adds the meat to the skeleton.
+
+JavaScript and the JS library jQuery are scripting languages that add functionality to a webpage. JS is used to work with the text and data a user may input on a web page, or it can pull in other info. JS is what makes our flesh covered skeleton dance.
+
+The DOM API allows for changes to the web page. Adding or removing content based on events, etc. This metaphor has gotten away from me.
```
Comment (bwreid) [inline readme.md:28]: This is mostly right. The key point is that jQuery/JS allows you to manipulate the DOM -- not the direct HTML file.

**Comment by bwreid** [comment]
Overall, nice job. Take a look at my comments and let me know if you have any questions. :+1:
